serverless.yml

```diff
@@ -27,6 +27,11 @@ functions:
       GOOGLE_APPLICATION_CREDENTIALS: firebase-admin-service-account.json
       SLACK_TOKEN: ${env:SLACK_TOKEN}
       IS_PROD: ${env:IS_PROD}
+  users:
+    handler: src/handler.handlerUsers
+    environment:
+      SLACK_TOKEN: ${env:SLACK_TOKEN}
```

Comment:
For this you can use secret manager: allows you to update without redeploying the lambda and avoid putting the secrets in the lambda ENV vars (which are available in plain text)
There are some usage examples in Kotlin in XDD, but I guess [similar SDKs](https://docs.aws.amazon.com/code-samples/latest/catalog/javascript-secrets-secrets_getsecretvalue.js.html) are available for JS/TS?